scripts/bumpVersions.ts

```diff
@@ -0,0 +1,28 @@
+import { promisify } from "util";
```

Comment:
👍 on renaming the file. The reason why we stage the change is because this script is run during the [npm version lifecycle](https://docs.npmjs.com/cli/v8/commands/npm-version#description). Short version is that running `npm version` will bump the npm package version, stage the `package{-lock}.json` files, then run the `version` script, then commit all staged files. This way, our NuGet package version is _always_ in sync w/ the npm package and both get version bumped in the same commit.

I'll add that reasoning to the script itself.